pkgs/os-specific/linux/flashbench/default.nix

```diff
@@ -1,27 +1,32 @@
-{ lib, stdenv, fetchgit }:
+{ lib, stdenv, fetchFromGitHub }:
 
 stdenv.mkDerivation {
   pname = "flashbench";
-  version = "2012-06-06";
+  version = "2020-01-23";
```

Comment:
```suggestion
  version = "unstable-2020-01-23";
```

Reply:
Thank you, I implemented your suggestion.